### ISSUE TITLE
Initial value?

### ISSUE THREAD

**Issue by amilich** [BitPhinix/slate-yjs#24]
I see the initial value inside Client is set to `[]`. How is the editor initialized with a non-empty array initial value? I'm seeing `Uncaught Error: Cannot get the start point in the node at path [] because it has no start text node.` with a basic setup.

**Comment by amilich** [comment]
Working off of https://github.com/BitPhinix/slate-yjs-example/tree/master/src

**Comment by BitPhinix** [comment]
Hi @amilich,

I'm really busy at the moment so it might take a while until I can take a look at this. As a "workaround" you can use something along the lines of

```
[{text: ""}]
```

as the initial state.

**Comment by amilich** [comment]
> Hi @amilich,
> 
> I'm really busy at the moment so it might take a while until I can take a look at this. As a "workaround" you can use something along the lines of
> 
> ```
> [{text: ""}]
> ```
> 
> as the initial state.

Got it - thanks for getting back to me during a busy time. I tried setting text to empty string, but now see 
```
react_devtools_backend.js:2273 TypeError: path 0 does not match doc []
    at iterate (index.js:15)
    at Array.reduce (<anonymous>)
    at getTarget (index.js:19)
    at insertText (insertText.js:10)
    at applySlateOp (index.js:19)
    at Array.reduce (<anonymous>)
    at applySlateOps (index.js:33)
    at yjsEd.tsx:23
    at transact (yjs.mjs:2098)
    at Doc.transact (yjs.mjs:364)
    at Object.applySlateOps (yjsEd.tsx:22)
    at Object.e.onChange (yjsEd.tsx:67)
    at index.es.js:5013 {type: "insert_text", path: Array(1), offset: 0, text: "w"} []
```
which isn't unexpected given that the YJS doc is initialized as empty.

Any idea why this initialization is causing trouble?

**Comment by amilich** [comment]
I ended up trying to port the code in server.ts to run in the browser, and I see:
```
react_devtools_backend.js:2273 Error: Unexpected content type in insert operation
    at yjs.mjs:2976
    at Array.forEach (<anonymous>)
    at typeListInsertGenericsAfter (yjs.mjs:2954)
    at typeListInsertGenerics (yjs.mjs:2995)
    at yjs.mjs:3264
    at transact (yjs.mjs:2098)
    at YArray.insert (yjs.mjs:3263)
    at toSyncDoc (convert.js:41)
```

any knowledge of why? I've tried inserting into the Y.Array and that seems fine, but calling toSyncDoc fails.

**Comment by amilich** [comment]
Was able to figure it out - seems like adding initial content to the document is possible, but only after the component has mounted - otherwise, the YJS doc and slate `value` may be out of sync.

**Comment by BitPhinix** [comment]
Hi @amilich,

This should be solved with the 1.0.0 release. Slate-yjs will now overwrite the slate value with the content inside the yjs shared type on load.
